content/changelog-stable-old/index.html.haml

```diff
@@ -0,0 +1,28 @@
+---
+layout: changelog
+title: Archived LTS Changelog
+show_ratings: false
+has_rss: false
+actions:
+  <a class="app-button app-button--primary app-mobile-hide" href="/changelog-stable">
+    <ion-icon name="arrow-back-circle-outline"></ion-icon>
+    Back to Recent LTS Changelog
+  </a>
+---
+
+.ratings
+  - # Load older changelogs (all except the most recent 25, ordered correctly)
+  - archived_changelogs = site.changelogs[:lts].reverse.drop(25)
+  - archived_changelogs.each do |release|
+    %div
+      = partial('../_partials/release-header.html.haml', :release => release, :url => 'changelog-stable-old')
+      - if release.changes
+        %div.app-releases__list__items
+          = partial('../_partials/changelog-changes.html.haml', :changes => release.changes)
+      - else
+        %p No notable changes for this release.
+
+.app-banner
+  This is the changelog archive. Changelog for recent releases can be found
```

Comment:
```suggestion
  This is the changelog archive. Changelogs for recent releases can be found
```
Adjust plural usage as there are multiple changelogs.

Reply:
This is the very opposite of what @krisstern suggested in their previous review and I tend to agree:
* changelog = document containing info about multiple releases
* changelog entry = part of changelog corresponding to a single release

Reply:
```suggestion
  This is the changelog archive. The changelog for recent releases can be found
```
Ah sorry about that. In that case it should have something in front of it like this.

Reply:
I have updated every suggestion Sir @kmartens27